### PR TITLE
[homematic] Update README.md

### PR DESCRIPTION
**Pull request by Josar**
Add CCU Firewall config.

### CODE REVIEW

**Comment by wborn** [comment]
It greatly would help non-German speakers if you provide the menu options in English. ;-)

File: addons/binding/org.openhab.binding.homematic/README.md [hunk 1]
```diff
@@ -1,8 +1,30 @@
 # Homematic Binding
 
 This is the binding for the [eQ-3 Homematic Solution](http://www.eq-3.de/).
-This binding allows you to integrate, view, control and configure all Homematic devices in Eclipse SmartHome.
+This binding allows you to integrate, view, control and configure all Homematic devices in OpenHAB.
```
Comment (wborn) [inline addons/binding/org.openhab.binding.homematic/README.md]: It's "openHAB":

```suggestion
This binding allows you to integrate, view, control and configure all Homematic devices in openHAB.
```

File: addons/binding/org.openhab.binding.homematic/README.md [hunk 2]
```diff
@@ -1,8 +1,30 @@
...
 
+## Configuration of the CCU
```
Comment (wborn) [inline addons/binding/org.openhab.binding.homematic/README.md]: Some Markdown renderers have issues when there is no new line after a header:

```suggestion
## Configuration of the CCU

```

File: addons/binding/org.openhab.binding.homematic/README.md [hunk 3]
```diff
@@ -1,8 +1,30 @@
...
+Also the IP of the device running OpenHab has to be set to the list of "IP addresses for restricted access".
+
+Also under `Home page > Settings > Control panel` with the menu `Security` the option `Authentication`has to be disabled as the binding does not support the configuration of `username` and `password`for the XML-RPC API.
+
+If this is not done the binding won't be able to connect to the CCU and the CCU Thing will stay uninitialized and sets a timeout exception:
+
+```
+xxx-xx-xx xx:xx:xx.xxx [hingStatusInfoChangedEvent] - - 'homematic:bridge:xxx' changed from INITIALIZING to OFFLINE (COMMUNICATION_ERROR): java.net.SocketTimeoutException: Connect Timeout
+```
 ## Supported Bridges
```
Comment (wborn) [inline addons/binding/org.openhab.binding.homematic/README.md]: Some Markdown renderers have issues when there is no new line before a header:

```suggestion

## Supported Bridges
```

File: addons/binding/org.openhab.binding.homematic/README.md [hunk 4]
```diff
@@ -1,8 +1,30 @@
...
+
+´´´
+2000;
+2001;
+2010;
+8701;
+9292;
+´´´
+
+Also the IP of the device running OpenHab has to be set to the list of "IP addresses for restricted access".
```
Comment (wborn) [inline addons/binding/org.openhab.binding.homematic/README.md]: It's "openHAB":

```suggestion
Also the IP of the device running openHAB has to be set to the list of "IP addresses for restricted access".
```

File: addons/binding/org.openhab.binding.homematic/README.md [hunk 5]
```diff
@@ -1,7 +1,32 @@
...
+
+´´´
+2000;
+2001;
+2010;
+8701;
+9292;
+´´´
+
+Also the IP of the device running openHab has to be set to the list of "IP addresses for restricted access".
```
Comment (wborn) [inline addons/binding/org.openhab.binding.homematic/README.md]: You've partially solved the casing issue. :wink: 

```suggestion
Also the IP of the device running openHAB has to be set to the list of "IP addresses for restricted access".
```

File: addons/binding/org.openhab.binding.homematic/README.md [hunk 6]
```diff
@@ -1,7 +1,32 @@
 # Homematic Binding
 
 This is the binding for the [eQ-3 Homematic Solution](http://www.eq-3.de/).
-This binding allows you to integrate, view, control and configure all Homematic devices in Eclipse SmartHome.
+This binding allows you to integrate, view, control and configure all Homematic devices in openHAB.
+
+## Configuration of the CCU
+
+Under `Home page > Settings > Control panel` with the menu `Configure Firewall` the Firewall configurations have to be adjusted.
+The CCU has to be configured to have "XML-RPC" set to "Full Acces" or "Restricted access".
```
Comment (wborn) [inline addons/binding/org.openhab.binding.homematic/README.md]: I think there's an "s" missing here:

```suggestion
The CCU has to be configured to have "XML-RPC" set to "Full Access" or "Restricted access".
```

File: addons/binding/org.openhab.binding.homematic/README.md [hunk 7]
```diff
@@ -1,7 +1,32 @@
...
+Also the "Remote Homematic-Script API" has to be set to "Full Acces" or "Restricted access".
```
Comment (wborn) [inline addons/binding/org.openhab.binding.homematic/README.md]: I think there's an "s" missing here:

```suggestion
Also the "Remote Homematic-Script API" has to be set to "Full Access" or "Restricted access".
```

**Comment by cweitkamp** [comment]
@davidgraeff @wborn Can you verify that everything is fine now? Thanks.

**Comment by Josar** [comment]
@wborn hope this is allright now.

**Comment by wborn** [comment]
Thanks a lot @Josar !

**Pull request merged**